<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-05T16:18:42 IST" name="smokeSuite" finished-at="2024-07-05T16:18:45 IST" duration-ms="3151">
    <groups>
    </groups>
    <test started-at="2024-07-05T16:18:42 IST" name="Notdefined" finished-at="2024-07-05T16:18:45 IST" duration-ms="3151">
      <class name="com.qa.Test.LoginTest">
        <test-method is-config="true" signature="setUpTest(java.lang.String)[pri:0, instance:com.qa.Test.LoginTest@3eb738bb]" started-at="2024-07-05T16:18:42 IST" name="setUpTest" finished-at="2024-07-05T16:18:45 IST" duration-ms="2853" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyLoginPageTitleTest()[pri:1, instance:com.qa.Test.LoginTest@3eb738bb]" started-at="2024-07-05T16:18:45 IST" name="verifyLoginPageTitleTest" finished-at="2024-07-05T16:18:45 IST" duration-ms="29" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [actual title from page] but found [Google]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [actual title from page] but found [Google]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.Test.LoginTest.verifyLoginPageTitleTest(LoginTest.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageTitleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.Test.LoginTest@3eb738bb]" started-at="2024-07-05T16:18:45 IST" name="tearDown" finished-at="2024-07-05T16:18:45 IST" duration-ms="225" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.Test.LoginTest -->
    </test> <!-- Notdefined -->
  </suite> <!-- smokeSuite -->
</testng-results>
